attic/ContentHandler.java

```diff
@@ -0,0 +1,33 @@
+package bio.terra.cloudfunctions.common;
```

Comment:
I'm not sure that this is the right approach.  Doesn't this mean that we're coupling how we handle a file's data to it's content type?  It's true that currently that gzip vs. json is one thing that delineates testrunner vs delta layer.  But it's possible that delta layer could start using gzip files in the future.

Reply:
This code is in the `attic` folder so it will not be compiled. For a general approach, I prefer just using the event and context objects to determine the cloud function type (`deltalayer` vs `testrunner` vs anything else). I think `App` is sufficient for this purpose.

Reply:
Oh, I see.  Sorry for not noticing that!  I understand that you may want to use the code in the `attic` folder in the future, but what about just publishing that code to another branch called `attic` and keeping it there rather than `main`?  I feel like that's a cleaner solution than having a folder with java files that aren't on the source path, but if you feel really strongly.  I could be persuaded.